scripts/item-descriptions.lua

```diff
@@ -0,0 +1,75 @@
+-- Holds custom descriptions for view-item-info
+-- By PeridexisErrant
+
+print("This is a content library; calling it does nothing.")
```

Comment:
```Lua
if not moduleMode then
 print("scripts/item-descriptions.lua is a content library; calling it does nothing.")
end
```

That will correct the ```script_environment``` issue and still print a helpful warning if called by the user. I also recommend saying the name of the file just in case it's triggered by something else. I'm tempted to suggest throwing an error so they see a stack trace but that's persnickety even by my standards.